Upstream: sent to mailing list,
          https://www.colm.net/pipermail/ragel/2017-May/001714.html

From e5dc4e520f21c1abec4b276aed0476397b471d40 Mon Sep 17 00:00:00 2001
From: Heiko Becker <heirecka@exherbo.org>
Date: Fri, 5 May 2017 14:17:00 +0200
Subject: [PATCH] Don't hard-code gcc

It's already searched for via AC_PROG_CC, so gcc only needs to be
replaced with CC. Helpful if gcc has a prefix based on the
architecture, for example.
---
 configure.ac | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index b86da8c..e865ce9 100644
--- a/configure.ac
+++ b/configure.ac
@@ -117,8 +117,8 @@ cat > conftest.m <<EOF
 int main() { return 0; }
 EOF
 GOBJC=""
-if gcc -x objective-c conftest.m -o conftest.bin 2>/dev/null; then
-	GOBJC="gcc -x objective-c"
+if "$CC" -x objective-c conftest.m -o conftest.bin 2>/dev/null; then
+	GOBJC="$CC -x objective-c"
 	AC_MSG_RESULT([yes])
 else
 	AC_MSG_RESULT([no])
-- 
2.12.2

